db/migrate/20160224155909_add_registered_at_to_subscription_facet.rb

```diff
@@ -0,0 +1,9 @@
+class AddRegisteredAtToSubscriptionFacet < ActiveRecord::Migration
+  def up
+    add_column(:katello_subscription_facets, :registered_at, :datetime, :null => false, :default => Time.now)
```

Comment:
The only odd thing that might happen is that new rows get added and we forget to set registered_at to the time the migration was run (instead of the time the row gets added). If we can live with that, then that's fine.